components/Section/SocialMediaSection.tsx

```diff
@@ -59,6 +61,16 @@ export default function SocialMediaSection({ bgColor = '' }) {
             <GitHubIcon style={iconStyle} />
           </a>
         </Grid>
+        <Grid item xs={4} md={1}>
```

Comment:
Instagram can be removed from the SocialMediaSection for now because we don't have a link yet